### PR TITLE
Makarii hlyva

### PR DESCRIPTION
**Pull request by mojoym**
HW4 is ready

### CODE REVIEW

File: MakariiHlyva/HW3/Practical Task1/main.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#---------------Work with string of Zen of Python file----------------------------
+
+file = open("python.txt", "r")
+data_from_file = file.read()
+
+# Find quantity of different word is file
+quantity_of_better = data_from_file.find('better')
```
Comment (kolyasalubov) [inline MakariiHlyva/HW3/Practical Task1/main.py:7]: method find() return index of element
you need to use method count()

File: MakariiHlyva/HW3/Practical Task2/main.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#-------------------Make some operations with int-type number--------------------------
+
```
Comment (kolyasalubov) [inline MakariiHlyva/HW3/Practical Task2/main.py:2]: good job

File: MakariiHlyva/HW3/Practical Task3/main.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#----------------Interchange the values ​​of two variables without using the third variable-------------------
+
```
Comment (kolyasalubov) [inline MakariiHlyva/HW3/Practical Task3/main.py:2]: good job

File: MakariiHlyva/HW4/main.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
+#-------------Calculate factorial of natural number, which will be entered by user, without using recurcion----------------
+
```
Comment (kolyasalubov) [inline MakariiHlyva/HW4/main.py:2]: try to implement this task without import math

File: MakariiHlyva/HW3/Practical Task1/main.py (hunk 1 above)
Comment (mojoym) [reply MakariiHlyva/HW3/Practical Task1/main.py:7]: ok, thanks))

File: MakariiHlyva/HW4/main.py (hunk 4 above)
Comment (mojoym) [reply MakariiHlyva/HW4/main.py:2]: may I use loop in this task?

**Pull request closed without merging**